catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -159,7 +159,7 @@ private void parsePermissionsFromString(List<String> permStrings,
 
     private PolicyResponse getWritePolicy(Metacard input, Map<String, Serializable> properties) {
         HashMap<String, Set<String>> operationPolicy = new HashMap<>();
-        if (Requests.isLocal(properties) && RegistryUtility.isRegistryMetacard(input)) {
+        if (Requests.isLocal(properties) && input != null && RegistryUtility.isRegistryMetacard(input)) {
```

Comment:
@shaundmorris When an update occurs, the `CatalogFrameworkImpl` tries to map the new metacard to the old metacard. In the case of an `id` update, it tries to use the `id`. In the case that the `id` is null (which for a new metacard it could be), it fails to do the matching and passes a `null` metacard to the policy plugins. Somehow the updates still work, but I'm not well-versed in the update logic.